src/backend/joanie/payment/models.py

```diff
@@ -61,11 +61,11 @@ class Invoice(BaseModel):
         null=False,
         blank=False,
     )
-    recipient_name = models.CharField(
-        _("invoice recipient"), max_length=40, null=False, blank=False
-    )
-    recipient_address = models.TextField(
-        _("invoice address"), max_length=255, null=False, blank=False
+    recipient_address = models.ForeignKey(
+        to="core.Address",
+        verbose_name=_("invoice address"),
+        related_name="+",
```

Comment:
Me neither :sweat_smile:

Reply:
This is a Django trick to have no related name in the targeted model.

> If you’d prefer Django not to create a backwards relation, set related_name to '+' or end it with '+'. For example, this will ensure that the User model won’t have a backwards relation to this model:
```python
user = models.ForeignKey(
    User,
    on_delete=models.CASCADE,
    related_name="+",
)
```